packages/webapp/src/utils/helpers.js

```diff
@@ -445,6 +445,25 @@ const getHumanReadableFileSize = (fileSizeInBytes, maximumDecimalPlaces = 2) =>
   return `${parseFloat(humanReadableFileSize.toFixed(parseInt(maximumDecimalPlaces)))} ${units}`
 }
 
+const getValidation = (metricValidation, href) => {
```

Comment:
Can you change the function name to be more verbose/descriptive, getValidation is too generic (as I know it was pulled straight from the uploadMetric handler where it would be fine, but in the helpers object it needs to be more descriptive